### PR TITLE
Zanderz/core 2064

### PR DESCRIPTION
**Pull request by zanderz**

### CODE REVIEW

File: go/libkb/upgrade_instructions.go [hunk 1]
```diff
@@ -58,6 +58,16 @@ func darwinUpgradeInstructions() {
...
 
+func windowsUpgradeInstructions() {
+	serverURI := ProductionServerURI
+	if DefaultRunMode == DevelRunMode {
+		serverURI = DevelServerURI
+	} else if DefaultRunMode == StagingRunMode {
+		serverURI = StagingServerURI
+	}
+
+	G.Log.Warning("To upgrade, download the latest Keybase installer from " + serverURI)
```
Comment (maxtaco) [inline go/libkb/upgrade_instructions.go]: is this a direct link to the page, or just the home page? Can you provide direct link to the install page?  Also, we don't need to get this right for staging or devel.

Comment (zanderz) [inline go/libkb/upgrade_instructions.go]: That is just keybase.io. If we want to hardcode in an URL, what should it
be? I just figured it was easier and more flexible not to, and whoever is
smart enough to download it once could figure out where to update from.

Comment (maxtaco) [inline go/libkb/upgrade_instructions.go]: I guess we should plumb it through from the server via the API.

Comment (zanderz) [inline go/libkb/upgrade_instructions.go]: ok, who will do the server side? or should I try doing it?

Comment (maxtaco) [inline go/libkb/upgrade_instructions.go]: I will do it hopefully tonight.  Will follow up.

Comment (maxtaco) [inline go/libkb/upgrade_instructions.go]: PR here: https://github.com/keybase/client/pull/1415

**Comment by zanderz** [comment]
ok, I did try to merge in your changes and pushed it.

File: go/libkb/upgrade_instructions.go [hunk 2]
```diff
@@ -60,6 +60,11 @@ func darwinUpgradeInstructions(g *GlobalContext, upgradeURI string) {
 	// TODO: non-brew update instructions
 }
 
+func windowsUpgradeInstructions(g *GlobalContext, upgradeURI string) {
+
+
+	G.Log.Warning("To upgrade, download the latest Keybase installer from " + upgradeURI)
```
Comment (maxtaco) [inline go/libkb/upgrade_instructions.go]: use `g` rather than `G`

**Comment by maxtaco** [comment]
small fix

**Pull request merged**